src/test/java/org/opensrp/service/EventServiceTest.java

```diff
@@ -229,42 +229,31 @@ public void testAddEventShouldEvaluatePlan() {
 	public void testProcessOutOfArea() throws SQLException {
 		scripts.add("client.sql");
 		populateDatabase();
-		Event event = new Event().withEventType("Vaccination").withProviderId("tester111")
+		Event event = new Event().withEventType("Out of Area Service - Vaccination").withProviderId("tester111")
 		        .withLocationId("2242342-23dsfsdfds").withIdentifier(Client.ZEIR_ID, "218229-3");
-		Event outOfAreaEvent = eventService.processOutOfArea(event, username);
-		
-		assertNotNull(outOfAreaEvent);
-		assertNotNull(outOfAreaEvent.getDetails());
-		assertEquals(1, outOfAreaEvent.getDetails().size());
-		assertEquals("biddemo", outOfAreaEvent.getDetails().get("out_of_catchment_provider_id"));
-		assertEquals("42abc582-6658-488b-922e-7be500c070f3", outOfAreaEvent.getLocationId());
-		assertEquals("biddemo", outOfAreaEvent.getProviderId());
```

Comment:
is there a reason these assertions have been removed?

Reply:
Yes the implementation was incorrect. **Out of Area** event that was used to create the new Vaccination event was not supposed to include the details. The details were to be included in the new  *Vaccination* event that was created after processing the **Out of Area Service - Vaccination**  event. I refactored this in the code so the tests was failing.